automod/flagstore/flagstore_mem.go

```diff
@@ -51,8 +46,8 @@ func (s MemFlagStore) Remove(ctx context.Context, key string, flags []string) er
 	for _, f := range flags {
 		delete(m, f)
 	}
-	out := []string{}
-	for f, _ := range m {
+	var out []string
```

Comment:
This is almost always the better way to initialize a slice. This sets it to `nil` - with no "slice header" - which is exactly equivalent to `[]string{}` (which _does_ have a slice header) in almost all contexts, such as `len` and `append` and `for range`, etc.